terraform/modules/upload-service/bucket.tf

```diff
@@ -31,6 +31,17 @@ resource "aws_iam_policy" "upload_areas_submitter_access" {
             "Resource": [
                 "arn:aws:s3:::${aws_s3_bucket.upload_areas_bucket.bucket}"
             ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:Get*",
+                "s3:List*"
+            ],
+            "Resource": [
+                "arn:aws:s3:::org-humancellatlas-upload-staging/*",
```

Comment:
Move this to variable to get rid of reference to humancellatlas

Reply:
Self-code-review is a thing.